tests/Resources/Page/OCPDashboard/Monitoring/Grafana.robot

```diff
@@ -18,13 +18,3 @@ Go To Grafana Dashbord Search
     ...    Click Element    ${SLIDE_MENU_DROPDOWN}
     ...    AND
     ...    Click Element    ${GRAFANA_SEARCH_BUTTON}
```

Comment:
Could you move this file to tests/Resources/Page/ODH/Grafana/Grafana.resource  (using the .resource extension to avoid a robocop error)

Reply:
I am working on adding that file in this test [ODS-752](https://github.com/red-hat-data-services/ods-ci/pull/335). Can I update it in that test?

Reply:
Updated